packages/video_player/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -37,6 +39,10 @@
 import java.util.Map;
 
 final class VideoPlayer {
+  private static final float bandwidthFraction = .8f;
+  private static final int minDurationForQualityIncreaseMs = 1000;
+  private static final int minDurationForQualityDecreaseMs = 2000;
+  private static final int minDurationToRetainAfterDiscardMs = 2000;
```

Comment:
Could you explain how you arrived at these specific numbers?

Reply:
These numbers are a fraction of the default values in Exoplayer. I can change the code to use the default values (just remove and call empty factory) or add these metrics in VideoPlayerOptions and permit the users to handle these.

here the Exoplayer file ref (https://github.com/google/ExoPlayer/blob/release-v2/library/core/src/main/java/com/google/android/exoplayer2/trackselection/AdaptiveTrackSelection.java)